codegen/sr.rs

```diff
@@ -21,25 +21,19 @@ use proc_macro2::{Ident, Span, TokenStream};
 /// Returns the corresponding Rust type used in structured representation
 /// for the given operand kind in the SPIR-V JSON grammar.
 pub fn get_operand_type_sr_tokens(kind: &str) -> TokenStream {
-    if kind.starts_with("Id") {
-        quote! { spirv::Word }
-    } else if kind == "LiteralInteger" || kind == "LiteralExtInstInteger" {
-        quote! { u32 }
-    } else if kind == "LiteralSpecConstantOpInteger" {
-        quote! { spirv::Op }
-    } else if kind == "LiteralContextDependentNumber" {
-        panic!("this kind is not expected to be handled here")
-    } else if kind == "LiteralString" {
-        quote! { String }
-    } else if kind == "PairLiteralIntegerIdRef" {
-        quote! { (u32, spirv::Word) }
-    } else if kind == "PairIdRefLiteralInteger" {
-        quote! { (spirv::Word, u32) }
-    } else if kind == "PairIdRefIdRef" {
-        quote! { (spirv::Word, spirv::Word) }
-    } else {
-        let kind = Ident::new(kind, Span::call_site());
-        quote! { spirv::#kind }
+    match kind {
```

Comment:
This is just refining existing implementation and not related to the new feature we are implementing here, right? In the future, it would be nice to have separate PRs.